tests/test_dataset.py

```diff
@@ -26,6 +26,7 @@
 from datumaro.components.launcher import Launcher
 from datumaro.components.media import Image
 from datumaro.util.test_utils import TestDir, compare_datasets
+import datumaro.components.hl_ops as hlops
```

Comment:
Why change the spelling of the imported name? I think you should keep it as `hl_ops`.

Reply:
There is no specific reason. Ok, changed.